larray/core/array.py

```diff
@@ -5192,14 +5192,18 @@ def __int__(self):
     def __float__(self):
         return self.data.__float__()
 
-    def equals(self, other, nan_equals=False):
+    def equals(self, other, rtol=0, atol=0, nan_equals=False):
         """
         Compares self with another array and returns True if they have the same axes and elements, False otherwise.
 
         Parameters
         ----------
         other: LArray-like
             Input array. aslarray() is used on a non-LArray input.
+        rtol : float, optional
```

Comment:
int or float

Reply:
int? why int? 
When users work with arrays containing big integers?

Reply:
Yes. The unit of your arrays could be kg when you only care at the ton level, or whatever.